card_creator/vanilla/index.js

```diff
@@ -42,6 +42,8 @@ function createCard(card, main) {
     desc = desc.replaceAll("[x]", "");
 
     const classes = game.functions.getClasses();
+    classes.push("Neutral");
```

Comment:
Neutral cards were impossible to add using the vanilla card creator up until this point.
This bug was introduced in a patch added in 03a5b12